region/maridia/outer/Mt. Everest.json

```diff
@@ -4246,6 +4306,23 @@
         "In the model, that'll be represented by going 2 -> 9 -> 5 via two shinespark strats."
       ]
     },
+    {
+      "link": [9, 5],
+      "name": "Leave With Grapple Swing",
```

Comment:
Is there a leaveWithGrappleJump missing?
I use the blocks below the door to align with the sides, if that matters.

Reply:
It would be possible to leave with that condition but currently I don't think it would add anything new. Aside from Everest itself, the two rooms with `comeInWithGrappleJump` entrance condition are Main Street and Red Fish Room, and those also have entrance conditions for `comeInWithGrappleSwing` from the Everest location. Added a devNote now to explain.